### PR TITLE
CCSMB-1: Procedure for Standards Submission

### PR DESCRIPTION
**Pull request by tomodachi94**
This RFC defines the protocol for defining a standard.

### CODE REVIEW

File: Standards/CCSMB-1.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# CCSMB-1: Procedures for submitting a standard
+
+This RFC defines the protocol for defining and submitting a standard to the CCSMB.
+
+## File names and formatting
+
+Each RFC should be named `CCSMB-n.md`, where `n` is a number counting upwards.
+
+Each RFC should be a [GitHub-Flavored Markdown](https://github.github.com/gfm/) document with American English spelling.
```
Comment (SkyTheCodeMaster) [inline Standards/CCSMB-1.md:9]: The way it has always been in CC is British English spelling.

Comment (tomodachi94) [reply Standards/CCSMB-1.md:9]: Ah, let's do that then.
```suggestion
Each RFC should be a [GitHub-Flavoured Markdown](https://github.github.com/gfm/) document with British English spelling.
```

File: Standards/CCSMB-1.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# CCSMB-1: Procedures for submitting a standard
```
Comment (SkyTheCodeMaster) [inline Standards/CCSMB-1.md:1]: This should be capitalized according to standard English APA/Whatever rules
https://capitalizemytitle.com/#capitalizationrules

File: Standards/CCSMB-1.md (hunk 1 above)
Comment (SkyTheCodeMaster) [inline Standards/CCSMB-1.md]: The way it has always been in CC is British English spelling.

File: Standards/CCSMB-1.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+This RFC defines the protocol for defining and submitting a standard to the CCSMB.
+
+## File names and formatting
```
Comment (SkyTheCodeMaster) [inline Standards/CCSMB-1.md:5]: This should *maybe* be capitalized according to standard rules.

**Review by SkyTheCodeMaster — CHANGES_REQUESTED** [review]

File: Standards/CCSMB-1.md (hunk 1 above)
Comment (tomodachi94) [reply Standards/CCSMB-1.md]: Resolved in [`75376de`](https://github.com/CCSMB/Standards/pull/1/commits/75376de185bc0b3565e8a6c7fadc956ff14469a0)

File: Standards/CCSMB-1.md (hunk 3 above)
Comment (tomodachi94) [reply Standards/CCSMB-1.md:5]:
```suggestion
## File Naming and Formatting
```

File: Standards/CCSMB-1.md (hunk 2 above)
Comment (tomodachi94) [reply Standards/CCSMB-1.md:1]: Maybe reword it a bit too:
```suggestion
# CCSMB-1: Procedures for Standards Submission
```

**Review by SkyTheCodeMaster — APPROVED** [review]
Looks good to me, only issue I personally have is that I think it should be a little more verbose, but I don't have the writing skills to expand it. Other opinions required.

**Comment by tomodachi94** [comment]
Re-requesting review as there have been significant changes.

**Review by SkyTheCodeMaster — APPROVED** [review]
LGTM 👍

**Comment by Ocawesome101** [comment]
as someone who never uses british english spelling i disagree somewhat with that but i appreciate that it's for consistency.

perhaps the sample files should be in `/standards/CCSMB-n.lua` rather than `/Standards/CCSMB-n` - imo that makes a little more sense.

**Comment by tomodachi94** [comment]
> perhaps the sample files should be in `/standards/CCSMB-n.lua` rather than `/Standards/CCSMB-n` - imo that makes a little more sense.

To clarify, `/Standards/CCSMB-n` is a folder that contains `.lua` files. I'll make note of that.

cc @Ocawesome101

**Comment by MasonGulu** [comment]
Where would non `.lua` files go? For file format standards and the like.

**Comment by Ocawesome101** [comment]
> To clarify, `/Standards/CCSMB-n` is a folder that contains `.lua` files. I'll make note of that.

ah.  excellent.

**Comment by tomodachi94** [comment]
> Where would non `.lua` files go? For file format standards and the like.

Hmm, good question. They could either go in-line in the document or in the RFC's folder.

@MasonGulu

**Review by Ocawesome101 — APPROVED** [review]

**Comment by tomodachi94** [comment]
> Why not just put them in the same folder? I don't see why there needs to be a separation of .lua and other files.

Hmm, good idea. Added in [`f45afca`](https://github.com/CCSMB/Standards/pull/1/commits/f45afca5361b6deb381ab13a030438f6013e93cd).

**Review by MasonGulu — APPROVED** [review]
That works for me.
